File: Avaliacao/q1_python/Console.py
from Produto import Produto

class Console(Produto):
    def __init__(self, nome, preco, geracao, marca):
        super().__init__(nome, preco)
        self.geracao = geracao
        self.marca = marca

    def mostra_info(self):
        super().mostra_info()
        print(f"Geracao: {self.geracao}")
        print(f"Marca: {self.marca}")

    def jogar(self):
        print("Carregando o jogo...")
        print("Jogo iniciou")

File: Avaliacao/q1_java/src/main/java/capacitacao/core/Console.java
package capacitacao.core;

public class Console extends Produto{
    private String geracao;
    private String marca;

    public Console(String nome, double preco, String geracao, String marca) {
        super(nome, preco);
        this.geracao = geracao;
        this.marca = marca;   
    }

    @Override
    public void mostraInfo(){
        super.mostraInfo();
        System.out.println("Geracao: " + this.geracao);
        System.out.println("Marca: " + this.marca);
    }

    public void jogar(){
        System.out.println("Carregando o jogo...");
        System.out.println("Jogo iniciou");
    }
}
